tests/foreman/ui/test_reporttemplates.py

```diff
@@ -233,6 +234,31 @@ def test_positive_generate_registered_hosts_report(session, module_org, module_l
             assert res['Operating System'] == '{0} {1}'.format(os_name, os.major)
 
 
+@upgrade
+@tier2
+def test_positive_generate_subscriptions_report_json(session, module_org, module_loc):
+    """Use provided Subscriptions report, generate JSON
+
+    :id: b44d4cd8-a88e-47cf-9993-0bb871ac2c96
+
+    :expectedresults: The Subscriptions report is generated in JSON
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Medium
+    """
+    # generate Subscriptions report
+    with session:
+        file_path = session.reporttemplate.generate("Subscriptions",
+                                                    values={'output_format': 'JSON'})
+    with open(file_path) as json_file:
+        data = json.load(json_file)
+        assert len(data) > 0
```

Comment:
I suggest to de-indent that and further lines, so if assertion fails, we don't leave file opened (pytest will complain file was left opened, but other than that, I don't think it has any practical consequences; it's just good practice).